ui/index.html

```diff
@@ -23,7 +23,7 @@
 
         <!-- BODY -->
         <main id="index-container" class="oae-branding-container oae-main-content" role="main">
-            <div id="index-page-container" class="row"><!-- --></div>
+            <div id="index-page-container" tabindex="0" class="row"><!-- --></div>
```

Comment:
If the examples don't add `tabindex`, I'd be happy removing it as well. The extra tab stop becomes somewhat annoying when using the keyboard with the tabindex in place.